### PR TITLE
Adding C Function For Getting ValueType of a CoreIR Type

### PR DESCRIPTION
**Pull request by David-Durst**

### CODE REVIEW

File: include/coreir-c/coreir-values.h [hunk 1]
```diff
@@ -16,6 +16,7 @@ extern COREValue* COREValueInt(COREContext* c,int i);
 extern COREValue* COREValueString(COREContext* c,char* str);
 extern COREValue* COREValueBitVector(COREContext* c, int width, uint64_t val);
 extern COREValue* COREValueModule(COREContext* cc, COREModule* mod);
+extern COREValue* COREValueCoreIRType(COREContext* cc, COREType* mod);
```
Comment (leonardt) [inline include/coreir-c/coreir-values.h]: Can you rename `mod` -> `type` to match the implementation (and make the parameter name consistent with the function name)

Comment (David-Durst) [reply include/coreir-c/coreir-values.h]: Change has been made.

**Comment by David-Durst** [comment]
Made the request change.

**Comment by leonardt** [comment]
Merged, thanks!

**Pull request merged**